### DISCUSSION TITLE
Old hotkeys still working

### DISCUSSION THREAD

**Discussion by imbee10** [Flow-Launcher/Flow.Launcher#2229] [Q&A] [answered]
why i can still use old hot keys instead of new one? 
there are two appearances, my new one is Alt X, can use plugins but the second one ( Alt Sp - old ) doesn't. 
![iTaE5fzo7b](https://github.com/Flow-Launcher/Flow.Launcher/assets/139006047/5f3143f4-6150-4003-8ff1-ac1b9b7e9193)
![Ytk36r4BDC](https://github.com/Flow-Launcher/Flow.Launcher/assets/139006047/a9d8582a-35cb-4b22-9d17-c28cf17a2f9b)

**Comment by Garulf** [comment, marked as answer]
One of those launchers is NOT Flow Launcher but Power toys run. 

The one that says "Start Typing..." is Power Toys Run.

**Reply by imbee10** [reply]
i see it <3 Thanks
